<launch>
	<node name="RosAria" pkg="rosaria" type="RosAria" />
</launch>